internal/service/advancedclustertpf/schema.go

```diff
@@ -350,6 +313,38 @@ func resourceSchema(ctx context.Context) schema.Schema {
 				Delete: true,
 			}),
 		},
+		Blocks: map[string]schema.Block{
+			"labels": schema.SetNestedBlock{
```

Comment:
A bit curious of what will happen if api returns `labels/tags: []` when no `labels/tags` are defined in the config. Hopefully, stored the same way

Reply:
i think so